src/database.cpp

```diff
@@ -133,17 +137,23 @@ bool DatabasePrivate::updateDatabase()
     DatabaseModel current = currentModel;
 
     if (last == current) {
+#ifdef QT_DEBUG
         qDebug("Databse is up-to-date");
```

Comment:
Some messages (like that is an information) and should be shown even in release mode. Qt itself contains lot of this messages. Like QObject::connect on invalid signal/slot. 
Some others is for debug purpose and should be remove completly

Reply:
Qdebug not run in release mode. That what I know.

Reply:
No QDebug run on release mode too unless QT_NO_DEBUG macro is defined